great_expectations/execution_engine/sqlalchemy_batch_data.py

```diff
@@ -139,11 +139,6 @@ def __init__(
                 # mssql expects all temporary table names to have a prefix '#'
                 if engine.dialect.name.lower() == "mssql":
                     generated_table_name = f"#{generated_table_name}"
-                if engine.dialect.name.lower() == "bigquery":
-                    raise ValueError(
-                        "No BigQuery dataset specified.  Include bigquery_temp_table in "
-                        "batch_spec_passthrough or a specify a default dataset in engine url"
-                    )
```

Comment:
This error was preventing GE from creating bigquery tables without specifying `batch_spec_passthrough`.  Removing this error allows for `sqlalchemy_batch_data.py` to behave in the same way as CLI

Please see `great_expectations/cli/batch_request` line 268-274